.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Please capitalize the first word, like the other commits: https://github.com/BenWiederhake/FidgetCubeBot/commits/master

Please reference issues in a way that is machine-readable, this way the issue gets closed automatically. In this case, it should look exactly like this:

`Closes: #1`

I write this here because github *still* has no UI to review commit messages directly :/